src/Components/Cards/ProductCard.js

```diff
@@ -0,0 +1,100 @@
+import "./ProductCard.css";
+import { Link } from "react-router-dom";
+import { StarRatings } from "../index";
+
+import FavoriteSharpIcon from "@material-ui/icons/FavoriteSharp";
+
+export default function ProductCard({ item, reducerState, dispatch }) {
+  const checkIfFav = () => {
+    if (
+      reducerState.favourites.find((itemsInFav) => itemsInFav.id === item._id)
+    ) {
+      return true;
+    }
+    return false;
+  };
+
+  return (
+    <div className="productCardContainer">
+      <div className="prodImageWrapper">
+        <img className="prodCardImg" src={item.image} alt="pizza" />
+        <div className="addToFavContainer">
+          <label className="heartLabel">
+            <input
+              className="heartBtn"
+              type="checkbox"
+              checked={checkIfFav()}
+              onChange={() =>
+                dispatch({
+                  type: "TOGGLE_FAVOURITES",
+                  payload: {
+                    id: item._id,
+                    img: item.image,
+                    title: item.title,
+                    price: item.price,
+                    isVeg: item.isVeg,
+                    ratings: item.rating,
+                  },
+                })
+              }
+            />
+            <FavoriteSharpIcon
+              style={{ fontSize: "2.2rem" }}
+              classes={{ root: "heartIcon" }}
+            />
+          </label>
+        </div>
+      </div>
+      <div className="prodInfoWrapper">
+        <h1 className="prodCardTitle">{item.title}</h1>
+
+        <div className="productPriceWrapper">
+          <p className="prodCardPrice"> ₹{item.price}</p>
+          <div className="isVegContainer">
+            {item.isVeg ? (
```

Comment:
Don't add extra JSX like this instead do the conditionals in the `className` prop itself

```suggestion
<div className={` isVeg ${item.isVeg ? 'veg' : 'nonVeg' }`}></div>
```

Reply:
Removed extra JSX as per the suggestion